messages/env.compute.collaborator.add.md

```diff
@@ -0,0 +1,13 @@
+# summary
+
+Add heroku user to all compute environments.
+
+# description
+
+You must provide the '--heroku-user' required flag.
```

Comment:
Remove this sentence because it's assumed that users must provide a required flag.

Is there more info you can add here to expand a bit on the summary? For example, will the user know what "attach an add-on" means?   If not, can you briefly describe what it means?  Are there other reasons why you'd want to add a Heroku user as a collaborator?  Are there implications?  Requirements, such as the Heroku user must already exist?  Does this command depend on other commands having been executed?  (NOTE that I know very little about Functions/compute environments, so my questions probably don't make sense.  I'm just providing examples of stuff you _might_ want to put in the description. And if there's nothing more you can say that is helpful to the user, that's fine.  But if there IS something to make this command more self-explanatory, then add it here.)